<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterView</class>
 <widget class="QDialog" name="ParameterView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="m_btn_open">
       <property name="text">
        <string>打开相机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btn_close">
       <property name="text">
        <string>关闭相机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btn_live">
       <property name="text">
        <string>采集</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btn_pause">
       <property name="text">
        <string>暂停</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btn_stop">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="m_btn_roi_01">
         <property name="text">
          <string>创建区域1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btn_roi_02">
         <property name="text">
          <string>创建区域2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="m_btn_cancel_roi_01">
         <property name="text">
          <string>取消区域1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btn_cancel_roi_02">
         <property name="text">
          <string>取消区域2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="AngstrongDemo.qrc"/>
 </resources>
 <connections/>
</ui>
